### DISCUSSION TITLE
Github Copilot gave me up.

### DISCUSSION THREAD

**Discussion by farkon00** [community/community#19060] [Copilot Conversations]
Github Copilot yesterday(20.06.2022) promised to never ever give me up, but today(21.06.2022) it was what he did. 
As you can see on screenshot 1 yesterday i type `// Never gonna` and git hub copilot promised to not give me up.
Today i have seen notification from github copilot in VS Code, it was telling, that it wasnt properly configured, i tried to fix it, but it brought me to page from getting free 60-days trial of copilot and then payment of 10$/month. And now i dont get any suggestion from copilot. It gave me up!
![screenshot 1](https://user-images.githubusercontent.com/59134591/174857088-6dddeff3-7d13-4e34-8651-cd8be75413af.png)

**Comment by rauzerfabiano** [comment]
We fed the copilot and now we have to pay for him... really, he's given up on us.
